suricata.yaml.in

```diff
@@ -168,6 +168,19 @@ outputs:
             # payload-length: yes      # enable dumping payload length, including the gaps
             # packet: yes              # enable dumping of packet (without stream segments)
             # metadata: no             # enable inclusion of app layer metadata with alert. Default yes
+            # If you want metadata, use:
+            # metadata:
+              # Include the decoded application layer (ie. http, dns)
+              #app-layer: true
+              # Log the current state of the flow record.
+              #flow: true
+              #rule:
+                # Log the metadata field from the rule in a structured
+                # format.
+                #metadata: true
+                # Log the raw rule text.
+                #raw: false
+                # reference: no        # include reference information from the rule
```

Comment:
(I apologize for not thinking of that before, but) we should add `reference` info to [EVE advanced settings](https://docs.suricata.io/en/latest/output/eve/eve-json-output.html#alerts), and we should also update https://github.com/OISF/suricata/blob/master/doc/userguide/partials/eve-log.yaml#L77 to reflect what we have here.